examples/lambda/registrator.tf

```diff
@@ -0,0 +1,10 @@
+module "consul_lambda_registrator" {
+  source                       = "../../modules/lambda-registrator"
+  name                         = "${var.name}-lambda-registrator"
+  ecr_image_uri                = var.ecr_image_uri
+  consul_http_addr             = "http://${module.dev_consul_server.server_dns}:8500"
+  consul_extension_data_prefix = "/${var.name}"
+  subnet_ids                   = module.vpc.private_subnets
+  security_group_ids           = [module.vpc.default_security_group_id]
+  sync_frequency_in_minutes    = 1
```

Comment:
I dropped the sync frequency down so that any changes are picked up quickly. I noticed on start that it was taking the full sync period (10 minutes) for the lambda functions to show up in Consul because they were created before the registrator.. I guess I could add a `depends_on` for the lambda functions. But I think this is fine for the example in any case.